crates/ark-contracts/arkchain/src/orderbook.cairo

```diff
@@ -27,6 +27,16 @@ trait Orderbook<T> {
     fn execute_order(
         ref self: T, order_hash: felt252, execution_info: ExecutionInfo, sign_info: SignInfo
     );
+
+    // Views
+    // Get order type from order_hash
+    fn get_order_type(self: @T, order_hash: felt252) -> felt252;
+    // Get order status from order_hash
+    fn get_order_status(self: @T, order_hash: felt252) -> felt252;
+    // Get order from order_hash
+    fn get_order(self: @T, order_hash: felt252) -> OrderV1;
+    // get order hash from token_hash, obviously only work for a listing
+    fn get_order_hash(self: @T, token_hash: felt252) -> felt252;
```

Comment:
For the comments, it could be great to have rust-doc like documentation for the functions inside the trait definition.
Not mandatory for this PR, comment aside. :+1: